guides/source/tuning_perf_for_deployment.md

```diff
@@ -0,0 +1,158 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
+
+Tuning Performance for Deployment
```

Comment:
this guide seems to be 100% focused on web servers. Should we make it clear in the title? or would you prefer to have things related to background jobs or websockets also added in this doc?

Reply:
Good question on background jobs and websockets. I'm not sure the title should be longer. It feels a bit unwieldy as-is. I'll add a bit of text near the top mentioning the focus on web servers.

With background jobs and websockets, I suspect they'll need their own guide. Both are quite involved and depend on what implementation is behind them (e.g. AnyCable, SideKiq.) Web servers are different in that the default server is still pretty solid for larger-scale production deployment, so the guide doesn't need to start with "add something non-default, please" and then describe a variety of them. I'm also less qualified to write those guides.